src/Shapes.hs

```diff
@@ -809,6 +828,25 @@ lineTransformType = ("LineTransform", M.fromList
         ("name", (StrT, constValue $ StrV "defaultLine"))
     ])
 
+squareTransformType = ("SquareTransform", M.fromList
+    [
+        -- knowing that the "original" is centered at origin, still need to store?
+        ("x", (FloatT, x_sampler)), -- x and y as dx and dy in transform?
```

Comment:
Yes, treated like dx and dy (feel free to rename the properties if that makes it clearer)